File: plugins/de.ovgu.featureide.examples/featureide_examples/GPL-FH-Java/features/MSTKruskal/GPL/Vertex.java
package GPL;

import java.lang.Integer;
import java.util.LinkedList;
import java.util.Collections;
import java.util.Comparator;

// of Graph

  // *************************************************************************

public class Vertex {
    public  Vertex representative;
    public LinkedList members;

    public void display() {
        if ( representative == null )
            System.out.print( "Rep null " );
        else
            System.out.print( " Rep " + representative.getName() + " " );
        original();
    }
}

File: plugins/de.ovgu.featureide.examples/featureide_examples/GPL-FH-CSharp/features/StronglyConnected/Vertex.cs
//layer StronglyConnected;

using System.Collections;
//import java.util.Collections;
//import java.util.Comparator;

// of Graph

  // ***********************************************************************
   
public class Vertex {//refines
    public int finishTime;
    public int strongComponentNumber;
      
    public void display() {
        System.Console.Out.Write( " FinishTime -> " + finishTime + " SCCNo -> " 
                        + strongComponentNumber );
        original();
    }
}
